cmd/sharded-test-server/shard.go

```diff
@@ -83,11 +94,18 @@ func newShard(ctx context.Context, n int, args []string, servingCA *crypto.CA, h
 		fmt.Sprintf("--tls-private-key-file=%s", filepath.Join(workDirPath, fmt.Sprintf(".kcp-%d/apiserver.key", n))),
 		fmt.Sprintf("--secure-port=%d", 6444+n),
 		fmt.Sprintf("--logical-cluster-admin-kubeconfig=%s", filepath.Join(workDirPath, ".kcp/logical-cluster-admin.kubeconfig")),
+		fmt.Sprintf("--shard-client-cert-file=%s", shardClientCert),
+		fmt.Sprintf("--shard-client-key-file=%s", shardClientCertKey),
+		fmt.Sprintf("--shard-virtual-workspace-ca-file=%s", filepath.Join(workDirPath, ".kcp", "serving-ca.crt")),
 	)
 	if len(cacheServerConfigPath) > 0 {
 		args = append(args, fmt.Sprintf("--cache-server-kubeconfig-file=%s", cacheServerConfigPath))
 	}
 
+	if standaloneVW {
+		args = append(args, fmt.Sprintf("--shard-virtual-workspace-url=https://%s:%d", hostIP, 7444+n))
```

Comment:
nit: might be useful to close over these port additions to label them - `func virtualWorkspacePort(n int)`

Reply:
Done